os/net/lwip/src/core/ipv4/ip_frag.c

```diff
@@ -63,7 +63,7 @@
 #include <net/lwip/snmp.h>
 #include <net/lwip/stats.h>
 #include <net/lwip/ipv4/icmp.h>
-
+#include <arpa/inet.h>
```

Comment:
Why include the header file here, if the rest of the file does not use it? Isn't it better to include files where they are used directly?

Reply:
After adding Werror flag we were getting 'ntohs' and 'htons' related error to overcome that error we have added this header file.